src/ivoc/gifimage.cpp

```diff
@@ -162,7 +161,7 @@ static int LoadGIF(const char* fname, PICINFO* pinfo)
 {
     /* returns '1' if successful */
 
-    byte ch, ch1, *origptr;
```

Comment:
Because this is already defined here xD

Reply:
so it's not removing an unused variable, it's changing how something is shadowed?